### DISCUSSION TITLE
Dinner Tonight hardcoded url

### DISCUSSION THREAD

**Discussion by edk4971** [mealie-recipes/mealie#4786] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

It's currently several clicks to get to the recipe I'd like to cook for dinner tonight. 

### Please provide a concise description of the feature that would resolve your issue.

I request a hard-coded URL shortcut for "dinner tonight", something like $BASE_URL/dinner-tonight that can be bookmarked to quickly open the recipe card for tonight. Ideally also with a $BASE_URL/dinner-tonight?cook_mode url variable that opens it directly into cook mode view.

### Please consider and list out some caveats or tradeoffs made in your design decision

The existing API capabilities don't seem to cover this use case. Additionally, they aren't really meant for end-user via browser. This functionality could be replicated with the API, but it would take a lot of JSON digestion. Most non-power users are going to use a browser. Having a simple link they can save to quickly take them what they want would be handy.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Can you not just navigate to the meal planner?
https://demo.mealie.io/household/mealplan/planner/view

**Reply by edk4971** [reply]
Yes, however having a bookmark makes mealie more user friendly to the less technically sophisticated in my household. 

It also opens up options for some kiosk mode use cases. For example, having the proposed "cook view" webpage card built into a home assistant dashboard.

**Reply by michael-genson** [reply]
I'm confused, what bookmark are you looking for? Can you not bookmark the meal planner? https://demo.mealie.io/household/mealplan/planner/view

If you want to show only one day, instead of the whole week, you can change that setting:
![image](https://github.com/user-attachments/assets/75f8ce6a-0753-4e26-8e11-77e30622a2ba)

Then you get this when you go to that page:
![image](https://github.com/user-attachments/assets/310db605-e6be-41d5-954e-af78fc1e0d6b)

**Comment by ChipSkylark37** [comment]
I could see as wanting to just show 'todays' value to show on a dashboard, or just as a quick reference without having to adjust the 'days'. Instead of having a specific bookmark just for 'dinner-tonight' when changing the number of days to display, could that be exposed to the URL? So if you do:

https://URL/household/mealplan/planner/view/1 
It would show '1 day = today'

https://URL/household/mealplan/planner/view/7
It would show the next 7 days. 

Maybe its not displayed by default, but if you do the /# where # = number of days, it sets that number to display on page load? 

Personally, I like to see the next 7 days, but can see how someone would want a quick bookmark to see just today or a preset value without having to change the 'Days to Display'

Also, I know it shows the 'on page load' but is there a way to force a refresh after this change is made? 

![image](https://github.com/user-attachments/assets/d61750be-ce98-4a32-a613-b4880e0778be)
